apps/transaction/views/category_manager_view.py

```
@@ -0,0 +1,112 @@
+
+    def dispatch(self, request, *args, **kwargs):
+        self.object = self.get_object()
+        self._ensure_room_member()
+        return super().dispatch(request, *args, **kwargs)
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context.update(self._build_context(category_creation_form=kwargs.get("category_creation_form")))
+        return context
+
+    def post(self, request, *args, **kwargs):
```

Ruff (0.14.10)

Missing type annotation for `*args`

(ANN002)